### ISSUE TITLE
[Question] Can I test teal against local algod/purestake instead of algob-runtime?

### ISSUE THREAD

**Issue by bo-abstrlabs** [scale-it/algo-builder#586]

**Comment by vuvth** [comment]
You can run scripts use `algob run` to test your program(more easier than `goal` command in my experience) . Sadly we don't support mocha for test in this case.

**Comment by bo-abstrlabs** [comment]
Thanks! I recently discovered this approach too. Start a node with algob infra and test something with algob run. It would be helpful to other developers if you could provide one or two example of such kind.

**Comment by vuvth** [comment]
Thanks! Developers can check [examples](https://github.com/scale-it/algo-builder/tree/master/examples) on our repos. I think it provide the useful example.

**Comment by robert-zaremba** [comment]
@bo-abstrlabs do you have a suggestion where else we should add more documentation?

**Comment by bo-abstrlabs** [comment]
@robert-zaremba

Sorry for late response. I found this script example is quite helpful: https://github.com/scale-it/algo-builder/blob/master/examples/crowdfunding/scripts/transfer/claim.js

And I was able to write test against local algod by reference this script. Maybe it's worth mention in https://github.com/scale-it/algo-builder/tree/master/examples#interesting-test-suites.

If you have more resources to work on the script: it's a script that requires setup to use it as an integration test. It would be perfect to make an example script (possibly in shell) that setup and call with the script.
